### DISCUSSION TITLE
How can I run the animation just once?

### DISCUSSION THREAD

**Discussion by jolo-dev** [unocss/unocss#781] [Q&A] [answered]
Hi there,

I would like that the opacity transitions to `1` after `animate-fade-in-down` and runs just once.
I see that `[animate-fade-in-down=""]{animation:fade-in-down 1s linear infinite;}` runs it `infinite` and with `animate-count-1` makes sure that this animation runs just once. But my text disappears..

Anyone an idea how to I can transition from `opacity-0` to `opacity-1` after running `animate-fade-in-down`?
To visualise my problem.

https://unocss.antfu.me/?html=DwEwlgbgBAFgtAMwK4BsVQC4FMAeG4DGWAdtgE5QIq5QDOW1B%2BxA9sVlAIZoBcGZnYrTAYwbOABYADFIB8AKChRQkKAFtOCpUpXQWAB04ERATzhTMufAFYc6BAHcAjDK7EwG7Ik4gscMMRwICwOxG4enF4ELEikcE7hnn5OtFraSrEsBLS0itrAAPTgEGk6xVAGAMwW2HhwKADmlA7VFmpOpdoAKjAcAbQYghgVgb4axCBQAIIYLGpgBFAAwgDKK1AkDQFYAHR5OkWQnbrqAEyU1DhQAFZIA2AIZkSkWBS1%2BKd2FfrVsCwQrwAvAAiAwADikwM6Ok4%2B3SUDAhE4ZAARuIUCwGiw4FsMDAkCi4el3v5iL0yCIidoYGQsAgQTAMBh9LQeAUCrj8SidtE1AVMtlaPzWILgVSlIMyA0sBgQQB9FEoQQAazF8KgskKmjhhWKaV1R3kBpKRvKnBRtBYKCQ2CgaKZczg1igFIajPMUGoCHwNSshBI5G%2Bv0crTS4jGgkmAHaEUJBqQoDHkGgTFACHdZvMAF7m6hGw4lIA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdA-6kAJRAA&options=N4IgzgLgTglgxhEAuaBXApgXyA

**Comment by userquin** [comment, marked as answer]
@jolo-dev just remove the opacity-0

**Reply by jolo-dev** [reply]
@userquin 
I forgot to mention that I have an `animate-delay-1s` 🤦‍♂️
It shouldn't be visible in the beginning.

**Reply by jolo-dev** [reply]
My updated version and increased the delay to see it

https://unocss.antfu.me/?html=DwEwlgbgBAFgtAMwK4BsVQC4FMAeG4DGWAdtgE5QIq5QDOW1B%2BxA9sVlAIZoBcGZnYrTAYwbOABYADFIB8AKChRQkKAFtOCpUpXQWAB04ERATzhTMufAFYc6BAHcAjDK7EwG7Ik4gscMMRwICwOxG4enF4ELEikcE7hnn6%2BKJxm1rSJkX5OtFraSrEsBLS0itrAAPTgEPk6NVAGAMwW2HhwKADmlA4tFmpOddoAKjAcAbQYghiNgb4axCBQAIIYLGpgBFAAwgDKu1AknQFYAHTlOtWQQ7rqAEyU1DhQAFZIk2AIZkSkWBRt%2BDudka%2BhasBYED%2BAF4AEQGAAcUhhQx0nAuBSgYEInDIACNxCgWJ0WHBjhgYEhceiCgD-MQxmQRNTtDAyFgELCYBgMPpaDxKpUyRTcadompKkUSrQJawpTDmUopmROlgMLCAPq41LEADW8oxUFkVU06KqNXyZuu8kttWtDU4uNoLBQSGwUHx3PWcGsUEZnS55ig1AQ%2BFaVkIJHIILBjj6%2BXE80ESwA7ZihFNSFBU8g0CYoAR3msNgAvB3Ua1XWpAA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdA-6kAJRAA&options=N4IgzgLgTglgxhEAuaBXApgXyA

**Reply by userquin** [reply]
@jolo-dev apply `animate-mode-backwards` instead `opacity-0`: [playground](https://unocss.antfu.me/?html=DwEwlgbgBAFgtAMwK4BsVQC4FMAeG4DGWAdtgE5QIq5QDOW1B%2BxA9sVlAIZoBcGZnYrTAYwbOABYADFIB8AKChRQkKAFtOCpUpXRBYDdjhqWILHABGnAgGsA7pzIhamXPgCsOdAjsBGGVzEBpxGCJxmcGDEcCAsdsSBwUYELEikcL6JhuZmKJwAnhku%2BtlFWtpKaSwEtLSK2sAA9OAQ5TotUCwADgDMUq54cCgA5pR2ff1qvm3aACowHFG0GIIYndFmGsQgUACCGCxqYARQAMIAyudQJMNRWAB09TrNkDO66gBMlNQ4UABWSGWYAQhSIpCwFGwgw%2BXk6vX6MBYEAhAF4AETdAAcUjRMx0nCeFSgYEIjgs4hQLGGLDgtwwMCQFkJFSh%2BCiCzIImZ2hgZCwCHRMAwGC6tB4jUadIZFnuKTUjSqNVoCtYSrR3KUKzIwywGHRAH0LHliDZ1USoLImppCU0WuVba95A7Wk6OpwLLQWCgkNgoOThYc4O4oJzhkK4P1qAh8P1WYQSOQ4X0xhNyuJNoIdgB24lCFakKA55BofJQAiAg5HABe7uoTperSAA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHCjjUrAFxwA2itV7azeqxjUAtOrGpaAGkRxmEMdB20oqZLTIBdA-6kAJRAA&options=N4IgzgLgTglgxhEAuaBXApgXyA)

**Reply by jolo-dev** [reply]
Awesome. That works 🥳

**Reply by userquin** [reply]
@jolo-dev https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode
